.github/workflows/update-ubuntu-finch-dependencies.yaml

```diff
@@ -0,0 +1,71 @@
+name: update-ubuntu-dependencies
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do we want this to run after ever merge to main? Seems like we'd be creating another PR after every merge?

Reply:
`steps.changes.outputs.changed == 'true'`
NVM, I missed this condition, will ensure a new PR is created only if there is a change in version number.